sentry/src/main/java/io/sentry/SessionAdapter.java

```diff
@@ -73,7 +73,9 @@ public void write(JsonWriter writer, Session value) throws IOException {
     boolean hasInitAttrs = false;
     hasInitAttrs = initAttrs(writer, hasInitAttrs);
 
-    writer.name("release").value(value.getRelease());
+    if (value.getRelease() != null) {
```

Comment:
This would result in a broken session update (if it's null)

Reply:
side effect of https://github.com/getsentry/sentry-java/pull/1439/files#r625319745
should not be nullable